src/View/Helper/FormHelper.php

```diff
@@ -1356,8 +1356,8 @@ protected function enumOptions(string $enumClass): array
         assert(is_subclass_of($enumClass, BackedEnum::class));
 
         $values = [];
+        /** @var \BackedEnum $case */
```

Comment:
The `assert()` about should have sufficed to indicate this is a backedenum class but alas phpstan isn't smart enough.